src/components/app/App.js

```diff
@@ -2,21 +2,45 @@ import React, { Component } from 'react'
 import './App.css'
 
 import movieData from '../../data/movieData'
-import Movies from '../Movies/Movies'
+import Movies from '/Users/tramtram/turing_work/3mod/projects/2week/rancid-tomatillos/src/components/movies/Movies.js'
```

Comment:
This looks like a local import - so when I pull this down, this import is going to break your app on my machine. Make sure to use a relative import :)